### ISSUE TITLE
[Question]: 多卡微调提示Out of memory 

### ISSUE THREAD

**Issue by Print1n** [PaddlePaddle/PaddleNLP#8232]
### 请提出你的问题

单卡使用
    --per_device_eval_batch_size 32 \
    --per_device_train_batch_size  32 \
 启动参数：
python  finetune.py  \
    --device gpu \
    --logging_steps 10 \
    --save_steps 100 \
    --eval_steps 100 \
    --seed 42 \
    --model_name_or_path uie-m-base \
    --output_dir $finetuned_model \
    --train_path data/train.txt \
    --dev_path data/dev.txt  \
    --max_seq_length 512  \
    --per_device_eval_batch_size 32 \
    --per_device_train_batch_size  32 \
    --num_train_epochs 20 \
    --learning_rate 1e-5 \
    --label_names "start_positions" "end_positions" \
    --do_train \
    --do_eval \
    --do_export \
    --export_model_dir $finetuned_model \
    --overwrite_output_dir \
    --disable_tqdm True \
    --metric_for_best_model eval_f1 \
    --load_best_model_at_end  True \
    --save_total_limit 1
 大概占用26.5G显存
 ![](https://imges-article.oss-cn-hangzhou.aliyuncs.com/202404071717116.png)
 
 同样的数据使用多卡微调，2*22G显卡
     --per_device_eval_batch_size 32 \
    --per_device_train_batch_size  32 \
 启动参数：
 python -u -m paddle.distributed.launch --gpus "0,1" finetune.py \
    --device gpu \
    --logging_steps 10 \
    --save_steps 100 \
    --eval_steps 100 \
    --seed 42 \
    --model_name_or_path uie-m-base \
    --output_dir $finetuned_model \
    --train_path data/train.txt \
    --dev_path data/dev.txt  \
    --max_seq_length 512  \
    --per_device_eval_batch_size 32 \
    --per_device_train_batch_size  32 \
    --num_train_epochs 100 \
    --learning_rate 1e-5 \
    --do_train \
    --do_eval \
    --do_export \
    --export_model_dir $finetuned_model \
    --label_names "start_positions" "end_positions" \
    --overwrite_output_dir \
    --disable_tqdm True \
    --metric_for_best_model eval_f1 \
    --load_best_model_at_end  True \
    --save_total_limit 1 \
    提示Out of memory 
    
    Question：为什么单卡26.5G可以微调成功，而多卡2*22G=44G却不能微调了？如果想让这两张卡能够使用上面的参数启动，我该怎么配置了？

**Comment by w5688414** [comment]
uie只适配了数据并行，没有模型并行，可以把batch_size调小进行训练
